spk/kopia/Makefile

```diff
@@ -1,5 +1,5 @@
 SPK_NAME = kopia
-SPK_VERS = 0.17.0
+SPK_VERS = 0.18.2
 SPK_REV = 1
```

Comment:
Last thing, this number needs to increment with each published version.

Reply:
SPK_REV means revision of SPK_VERS or revision of package? 

Because this PR is first revision of 0.18.2 and second one of package :)

Thanks for clarification

Reply:
If you take a look at our wiki on [Makefile variables](https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables), it states:
> Package Revision number. Starts at `1`, goes up 1 for every subsequent release unrelated to SPK_VERS